### PR TITLE
RSS Block: support className attribute

### PR DESCRIPTION
**Pull request by Soean**
## Description
Adds the custom `className` attribute to the RSS block. 
Also orders the attributes in a alphabetical order.

### CODE REVIEW

File: packages/block-library/src/rss/index.php [hunk 1]
```diff
@@ -79,8 +79,17 @@ function render_block_core_rss( $attributes ) {
...
 
-	$classes           = 'grid' === $attributes['blockLayout'] ? ' is-grid columns-' . $attributes['columns'] : '';
-	$list_items_markup = "<ul class='wp-block-rss{$classes}'>{$list_items}</ul>";
+	$class = 'wp-block-rss';
+
+	if ( 'grid' === $attributes['blockLayout'] ) {
+		$class .= ' is-grid columns-' . $attributes['columns'];
+	}
+
+	if ( isset( $attributes['className'] ) ) {
```
Comment (jorgefilipecosta) [inline packages/block-library/src/rss/index.php]: I think we need to check for an empty className. Right now, if we write a custom className and then we delete the custom class name the class field on the dom gets an empty space.

Comment (noisysocks) [reply packages/block-library/src/rss/index.php]:
```suggestion
	if ( ! empty( $attributes['className'] ) ) {
```

**Comment by codebykat** [comment]
I had to rebase this because I was getting "The editor has encountered an unexpected error" for whatever reason. But after a rebase and applying the suggested change by @noisysocks it is working great!

Testing steps I followed:
1. Add an RSS block to a post
2. Add a URL and click "Use URL"
3. Add an Additional CSS Class under Advanced
4. Update the post and verify that the class name was added, along with wp-block-rss
5. Delete the class name, update the post and verify that the class name is set to only wp-block-rss (with no empty spaces in the DOM).

**Review by codebykat — CHANGES_REQUESTED** [review]
I had to rebase this because I was getting "The editor has encountered an unexpected error" for whatever reason. But after a rebase and applying the suggested change by @noisysocks it is working great!

Testing steps I followed:
1. Add an RSS block to a post
2. Add a URL and click "Use URL"
3. Add an Additional CSS Class under Advanced
4. Update the post and verify that the class name was added, along with wp-block-rss
5. Delete the class name, update the post and verify that the class name is set to only wp-block-rss (with no empty spaces in the DOM).

**Comment by Soean** [comment]
merged in #14533

**Pull request closed without merging**